fe/fe-core/src/main/java/org/apache/doris/catalog/TableIf.java

```diff
@@ -244,5 +244,9 @@ default boolean isManagedTable() {
     default long getLastUpdateTime() {
         return -1L;
     }
+
+    default long getDataSize() {
+        return 0;
+    }
```

Comment:
should not provide default implementation. all tableIf should impl it by themsleves

Reply:
There are too many table types to implement this method on every of them

Reply:
I will add a comment on this method as what `Command::run` has done